packages/cardpay-sdk/sdk/safes/base.ts

```diff
@@ -200,36 +200,50 @@ export default class Safes {
     return gasInToken;
   }
 
+  async sendTokens(txnHash: string): Promise<TransactionReceipt>;
   async sendTokens(
     safeAddress: string,
     tokenAddress: string,
     recipient: string,
     amount: string,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async sendTokens(
+    safeAddressOrTxnHash: string,
+    tokenAddress?: string,
+    recipient?: string,
+    amount?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    if (!tokenAddress) {
+      let txnHash = safeAddressOrTxnHash;
+      return waitUntilTransactionMined(this.layer2Web3, txnHash);
+    }
+    let safeAddress = safeAddressOrTxnHash;
+    let { nonce, onNonce, onTxnHash } = txnOptions ?? {};
     let from = contractOptions?.from ?? (await this.layer2Web3.eth.getAccounts())[0];
     let token = new this.layer2Web3.eth.Contract(ERC20ABI as AbiItem[], tokenAddress);
     let symbol = await token.methods.symbol().call();
     let safeBalance = new BN(await token.methods.balanceOf(safeAddress).call());
-    if (safeBalance.lt(new BN(amount))) {
+    if (safeBalance.lt(new BN(amount!))) {
```

Comment:
ditto with the comments above around using a test to assert this exists instead of using `!`'s